src/Employer/Employer.Web/Views/Part1/TrainingProvider/SelectTrainingProvider.cshtml

```diff
@@ -0,0 +1,109 @@
+@inject IOptions<ExternalLinksConfiguration> _externalLinks;
+
+@model SelectTrainingProviderViewModel
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-two-thirds">
+        <a asp-show="@Model.PageInfo.IsWizard" asp-route="@RouteNames.Title_Get" esfa-automation="link-back" class="govuk-back-link govuk-!-margin-top-0 govuk-!-margin-bottom-6">Back</a>
+        <partial name="@PartialNames.ValidationSummary" model='new ValidationSummaryViewModel { ModelState = ViewData.ModelState }' />
+        <partial name="@PartialNames.ReviewSummary" for="Review" />
+        <h1 class="govuk-heading-xl">
+            <span class="govuk-caption-l">@Model.Title</span>
+            Have you found a training provider?
+        </h1>
+        <form asp-route="@RouteNames.TrainingProvider_Select_Post" asp-route-wizard="@Model.PageInfo.IsWizard" novalidate>
+            <p class="govuk-body">
+                You will need to use a registered training provider who offers the apprenticeship training you choose.
+            </p>
+            <div class="govuk-form-group">
+                <div class="govuk-radios govuk-radios--conditional" data-module="radios">
+                    <div class="govuk-radios__item">
+                        <input asp-for="SelectTrainingProvider"
+                               class="govuk-radios__input"
+                               id="found_yes"
+                               type="radio"
+                               value="true"
+                               data-aria-controls="found_yes_conditional">
+                        <label class="govuk-label govuk-radios__label" for="found_yes_conditional">
+                            Yes
+                        </label>
+                    </div>
+                    <div class="govuk-radios__conditional" id="found_yes_conditional">
+                        <div class="govuk-!-margin-bottom-2">
+                            <noscript>
+                                <div class="govuk-form-group" id="non-js-search">
+                                    <span class="govuk-hint">
+                                        Enter the training provider reference number (UKPRN). You will be able to get this from your training provider The UKPRN must only be 8 digits (For example, 12345678)
+                                    </span>
+                                    <span esfa-validation-message-for="Ukprn" class="govuk-error-message"></span>
+                                    <input asp-for="Ukprn" type="text" maxlength="8" class="govuk-input govuk-input--width-10" />
+                                </div>
+                            </noscript>
+                            <div class="govuk-form-group js-hidden" id="js-search">
+                                <span class="govuk-hint">
+                                    Enter the training provider or reference number (UKPRN). <br>
+                                    UKPRN is 8 numbers. For example 12345678.
```

Comment:
on Line 40 we have used 8 digits and here 8 numbers.

Reply:
👍  Content is a bit of a mess. I'll catch up with Helen.